otterdog/adoptium.jsonnet

```
@@ -1,5 +1,29 @@
 local orgs = import 'vendor/otterdog-defaults/otterdog-defaults.libsonnet';
 
+local jdkName(name) =
+  if std.startsWith(name, "jdk") then 
+    "JDK%s" % [ std.substr(name, 3, std.length(name) - 3)]
+  else
+    name;
+
+local newMirrorRepo(repoName) = orgs.newRepo(repoName) {
+  jdk_name:: jdkName(repoName),
+  jdk_url:: "https://github.com/openjdk/%s" % [self.name],
+  allow_merge_commit: true,
+  allow_update_branch: false,
+  default_branch: "master",
+  delete_branch_on_merge: false,
+  dependabot_alerts_enabled: false,
+  has_issues: false,
+  has_projects: false,
+  has_wiki: false,
+  homepage: self.jdk_url,
```

Just to avoid any confusion about this being an OpenJDK repo...

```suggestion
  homepage: "https://adoptium.net",
```